packages/commons/src/events/ActionDocument.ts

```diff
@@ -42,6 +43,10 @@ export const ActionBase = z.object({
   createdAt: z.string().datetime(),
   createdBy: z.string(),
   createdByRole: z.string(),
+  createdBySignature: z
+    .string()
+    .nullish() // @TODO: Check where the null comes from
```

Comment:
should this TODO be completed on this PR? to me the comment doesnt say much

I think the same applies to the other TODO comments